build_tools/scripts/generate_release_index.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python3
+
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+"""Creates an HTML page for releases for `pip install --find-links` from GitHub releases."""
+# TODO(#10479) since we're generating this we might as well create a PEP 503
+# compliant index
```

Comment:
Does pip at least understand what this script generates now?

If it's basically https://llvm.github.io/torch-mlir/package-index/, I'd really like some CSS and structure/formatting as a human user ;P

Reply:
Yes. You can use it with `pip --find-links` It is not really intended for humans. This is what a PEP 503 compliant index looks like: https://pypi.org/simple/. It's not exactly for humans either :-)